packages/alea-frontend/pages/course-home/[courseId].tsx

```diff
@@ -203,6 +205,10 @@ const CourseHomePage: NextPage = () => {
               <QuizIcon fontSize="large" />
             </CourseComponentLink>
           )}
+          <CourseComponentLink href={`/homework/${courseId}`}>
```

Comment:
Let's hide this in production till the feature is ready:
``` 
process.env.NEXT_PUBLIC_SITE_VERSION !== 'production' && 
<CourseComponentLink href={`/homework/${courseId}`}>{t.homeworks}&nbsp;
            <AssignmentTurnedInIcon fontSize="large" />
          </CourseComponentLink>
```

Reply:
Fixed !!